modules/roles_profiles/manifests/profiles/openssh.pp

```diff
@@ -0,0 +1,71 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+class roles_profiles::profiles::openssh {
+
+    case $::operatingsystem {
+        'Windows': {
+        $programfiles     = $facts['programfiles']
+        $systemdrive      = $facts['systemdrive']
+        $programdata      = $facts['programdata']
+        $pwrshl_run_scrpt = lookup('pwrshl_run_scrpt')
+        $sshscrpt         = '"C:\Program Files\OpenSSH-Win64\install-sshd.ps1"'
+
+            defined_classes::pkg::win_zip_pkg { 'OpenSSH-Win64':
```

Comment:
This should be a class under win_packages that is then included here.

Reply:
Moved to its own module, and is now being called from the ssh profile.